resources/lib/database/db_base_sqlite.py

```diff
@@ -28,6 +28,19 @@
 # LIKE comparator use ASCII, the unicode chars are not comparable
 # ---------------------------------------------------------------------------
 
+def is_libsqlite3_threadsafe():
+    """
+    Check if libsqlite3 is threadsafe
+    """
+    try:
+        conn = sql.connect(":memory:")
```

Comment:
this project use `'` char for declaring strings so please change also all others